DigitalExcellence/dex-frontend
# Feature/389-implement-project-likes

## Description

With this pull request, the functionality to like projects will be available

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to a project overview page
2. Like a project
3. Refresh the page and see the like count will remain the same and the icon is solid.
4. Unlike the project and see the count go down and the icon is regular.
5. Go to a project detail page
6. Like the project
7. Close the modal and see the like count and icon are updated

## Link to issue
#389 

Closes: #389
